index.js

```diff
@@ -1,14 +1,16 @@
 const NEXT_CONTACT_BATCH_KEY = 'next_hubspot_contacts_url'
 const SYNC_LAST_COMPLETED_DATE_KEY = 'last_job_complete_day'
+const NEXT_DEAL_BATCH_KEY = 'next_hubspot_deals_url'
+const NEXT_COMPANY_BATCH_KEY = 'next_hubspot_companies_url'
 
-export const jobs = {
-   'Clear storage': async (_, { storage }) => {
-       await storage.del(NEXT_CONTACT_BATCH_KEY)
-       await storage.del(SYNC_LAST_COMPLETED_DATE_KEY)
-   }
+const jobs = {
+    'Clear storage': async (_, { storage }) => {
+        await storage.del(NEXT_CONTACT_BATCH_KEY)
+        await storage.del(SYNC_LAST_COMPLETED_DATE_KEY)
+    }
 }
 
-export async function setupPlugin({ config, global }) {
+async function setupPlugin({ config, global }) {
```

Comment:
why are we removing `export`? this should be here. all plugin functions should be exported

Reply:
Actually, I have exported it below at the end along with the other exports.